tests/utils_tests/test_datastructures.py

```diff
@@ -275,6 +275,12 @@ def test_create_with_invalid_key(self):
         with self.assertRaisesMessage(ValueError, msg):
             CaseInsensitiveMapping([(1, '2')])
 
+    def test_create_with_not_mappings(self):
+        data = {'Accept': 'application/json'}
+        for iterable in (tuple, list, ImmutableList):
```

Comment:
Please use `with self.subTest(...):` in the loop.

Reply:
I removed the test from the changeset